openspace.cfg

```diff
@@ -25,6 +25,7 @@ return {
     -- SGCTConfig = sgct.config.single{2560, 1440, shared=true, name="WV_OBS_SPOUT1"},
 
     -- SGCTConfig = "${CONFIG}/spout_output.xml",
+    -- SGCTConfig = "${CONFIG}/stereo.xml",
```

Comment:
No need to add this;  better to use `stereo="..."` in the `sgct.config.single` call

Reply:
It is true but I think it is easy for some of our partners wanting to test stereo on their machines as well for as developers. I vote to maintain it in the .cfg file. What do you think?